Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -0,0 +1,860 @@
+    {{3234, 0x4402}, "Power Blue Gold Rusher"},
+    {{3235, 0x0000}, "Shield Striker"},
+    {{3236, 0x0000}, "Sun Runner"},
+    {{3236, 0x4403}, "Legendary Sun Runner"},
+    {{3237, 0x0000}, "Sea Shadow"},
+    {{3237, 0x4402}, "Dark Sea Shadow"},
+    {{3238, 0x0000}, "Splatter Splasher"},
+    {{3238, 0x4402}, "Power Blue Splatter Splasher"},
+    {{3239, 0x0000}, "Soda Skimmer"},
+    {{3240, 0x0000}, "Barrel Blaster (Nintendo Only)"},
+    {{3240, 0x4402}, "Dark Barrel Blaster (Nintendo Only)"},
+    {{3239, 0x4402}, "Nitro Soda Skimmer"},
+    {{3241, 0x0000}, "Buzz Wing"},
+    {{3400, 0x0000}, "Fiesta"},
+    {{3400, 0x4515}, "Frightful Fiesta"},
+    {{3401, 0x0000}, "High Volt"},
+    {{3402, 0x0000}, "Splat"},
+    {{3402, 0x4502}, "Power Blue Splat"},
+    {{3406, 0x0000}, "Stormblade"},
+    {{3411, 0x0000}, "Smash Hit"},
+    {{3411, 0x4502}, "Steel Plated Smash Hit"},
+    {{3412, 0x0000}, "Spitfire"},
+    {{3412, 0x4502}, "Dark Spitfire"},
+    {{3413, 0x0000}, "Hurricane Jet Vac"},
+    {{3413, 0x4503}, "Legendary Hurricane Jet Vac"},
+    {{3414, 0x0000}, "Double Dare Trigger Happy"},
+    {{3414, 0x4502}, "Power Blue Double Dare Trigger Happy"},
+    {{3415, 0x0000}, "Super Shot Stealth Elf"},
+    {{3415, 0x4502}, "Dark Super Shot Stealth Elf"},
+    {{3416, 0x0000}, "Shark Shooter Terrafin"},
+    {{3417, 0x0000}, "Bone Bash Roller Brawl"},
+    {{3417, 0x4503}, "Legendary Bone Bash Roller Brawl"},
+    {{3420, 0x0000}, "Big Bubble Pop Fizz"},
+    {{3420, 0x450E}, "Birthday Bash Big Bubble Pop Fizz"},
+    {{3421, 0x0000}, "Lava Lance Eruptor"},
+    {{3422, 0x0000}, "Deep Dive Gill Grunt"},
+    {{3423, 0x0000}, "Turbo Charge Donkey Kong (Nintendo Only)"},
+    {{3423, 0x4502}, "Dark Turbo Charge Donkey Kong (Nintendo Only)"},
+    {{3424, 0x0000}, "Hammer Slam Bowser (Nintendo Only)"},
+    {{3424, 0x4502}, "Dark Hammer Slam Bowser (Nintendo Only)"},
+    {{3425, 0x0000}, "Dive-Clops"},
+    {{3425, 0x450E}, "Missile-Tow Dive-Clops"},
+    {{3426, 0x0000}, "Astroblast"},
+    {{3426, 0x4503}, "Legendary Astroblast"},
+    {{3427, 0x0000}, "Nightfall"},
+    {{3428, 0x0000}, "Thrillipede"},
+    {{3428, 0x450D}, "Eggcited Thrillipede"},
+    {{3500, 0x0000}, "Sky Trophy"},
+    {{3501, 0x0000}, "Land Trophy"},
+    {{3502, 0x0000}, "Sea Trophy"},
+    {{3503, 0x0000}, "Kaos Trophy"},
+};
+
+SkylanderPortalWindow::SkylanderPortalWindow(QWidget* parent) : QWidget(parent)
+{
+  setWindowTitle(tr("Skylanders Manager"));
+  setObjectName(QString::fromStdString("skylanders_manager"));
+  setMinimumSize(QSize(700, 200));
+
+  CreateMainWindow();
+
+  connect(&Settings::Instance(), &Settings::EmulationStateChanged, this,
+          &SkylanderPortalWindow::OnEmulationStateChanged);
+
+  installEventFilter(this);
+
+  OnEmulationStateChanged(Core::GetState());
+};
+
+SkylanderPortalWindow::~SkylanderPortalWindow()
+{
+}
+
+void SkylanderPortalWindow::CreateMainWindow()
+{
+  QVBoxLayout* mainLayout = new QVBoxLayout();
```

Comment:
```suggestion
  auto* mainLayout = new QVBoxLayout();
```

don't repeat yourself: the type already appears on the right hand side

also, the variable name should be written as `main_layout` to conform to our coding style